cmd/jimmsrv/main.go

```
@@ -207,30 +207,36 @@ func start(ctx context.Context, s *service.Service) error {
 		Handler:           jimmsvc,
 		ReadHeaderTimeout: time.Second * 5,
 	}
+
+	maxConccurentConncetions, _ := strconv.Atoi(os.Getenv("JIMM_SSH_MAX_CONCURRENT_CONNECTIONS"))
```

Spelling & why ignore err?

---

It was like this, I assume because the ssh server verifies this value and sets a default if it's out of bounds.

---

yep, kian is right